src/tune.h

```diff
@@ -34,7 +34,7 @@ using Range    = std::pair<int, int>;  // Option's min-max values
 using RangeFun = Range(int);
 
 // Default Range function, to calculate Option's min-max values
-inline Range default_range(int v) { return v > 0 ? Range(0, 2 * v) : Range(2 * v, 0); }
```

Comment:
contains changes that should probably not be merged.
Please rebase after the next merge wave, and squash in a single commit.